### DISCUSSION TITLE
images for md file are all of sudden showing up under main portion of my repository

### DISCUSSION THREAD

**Discussion by apsessoms** [community/community#68195] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello! I had a repo that used to look like this and very organized. 
![image](https://github.com/community/community/assets/99392512/5f1f1fe1-c7ad-4471-acaf-4ea480f732f3)

However, I created a new tutorial that is called Network Security Group and each individual image file uploaded underneath and now it looks cluttered and busy. Does anyone know why this is and could they help me fix this problem?

![image](https://github.com/community/community/assets/99392512/98d8681f-0876-4848-aa5b-2cf6bbacdde2)

**Comment by rantirules** [comment]
If the images are on the same level then they will appear like that otherwise create a folder like `src` or `assets` and put them all there and modify your links accordingly by using `folder_name/image-1.png` if you are using the images elsewhere.

**Reply by apsessoms** [reply, marked as answer]
Thank you! I will try this now.
